diagnostics/ENSO_MSE/shared/set_variables_AM4.py

```diff
@@ -0,0 +1,22 @@
+import os
+
```

Comment:
You can combine set_variables_AM4, set_variables_CESM, and set_variables_CMIP as suggested for ENSO_RWS [here](https://github.com/NOAA-GFDL/MDTF-diagnostics/pull/223#discussion_r645139889)